packages/financial-templates-lib/test/price-feed/CreatePriceFeed.js

```diff
@@ -14,15 +14,6 @@ const Timer = getContract("Timer");
 const Store = getContract("Store");
 const AddressWhitelist = getContract("AddressWhitelist");
 
-// Pull in contracts from contracts-node sourced from the across repo.
```

Comment:
I removed everything to do with across from the financial templates lib tests file for price feeds.